doc/developer-notes.md

```diff
@@ -1119,6 +1119,19 @@ to check a subtree directory for consistency with its upstream repository.
 
 The tool instructions also include a list of the subtrees managed by Bitcoin Core.
 
+To fully verify or update a subtree, add it as a remote:
+
+```sh
+git remote add --fetch libmultiprocess https://github.com/bitcoin-core/libmultiprocess.git
```

Comment:
No need to add the `--fetch` option. You added a `git fetch` below.

Reply:
You only need to add it as a remote once, but each subsequent update requires a fetch. I'll drop `--fetch` from the first command though.